src/layout/RepeatingGroup/Summary2/RepeatingGroupSummary.tsx

```diff
@@ -40,7 +40,7 @@ export const RepeatingGroupSummary = ({
   if (rows.length === 0) {
     return (
       <SingleValueSummary
-        title={title}
+        title={<Lang id={title} />}
```

Comment:
You should use the `node`-prop to make sure text-resources resolve correctly. Same for the other one.

```suggestion
        title={<Lang id={title} node={componentNode} />}
```